frontend/server/src/Controllers/Identity.php

```diff
@@ -729,7 +728,7 @@ function () use ($identity, $user) {
     /**
      * Returns the profile of the identity given
      *
-     * @return array{birth_date: null, classname: string, gender: null, graduation_date: null, gravatar_92: null, hide_problem_tags: bool, scholar_degree: null, verified: bool, country: null|string, country_id: null|string, is_private: true, locale: string, name: null|string, preferred_language: null, school: null|string, school_id: int|null, state: null|string, state_id: null|string, username: null|string}
+     * @return array{birth_date: null|string, classname: null|string, country: null|string, country_id: null|string, gender: null|string, graduation_date: null|string, gravatar_92: null, hide_problem_tags: bool|null, is_private: true, locale: string, name: null|string, preferred_language: null, scholar_degree: null|string, school: null|string, school_id: int|null, state: null|string, state_id: null|string, username: null|string, verified: bool|null}
```

Comment:
```suggestion
     * @return array{birth_date: null|string, classname: null|string, country: null|string, country_id: null|string, gender: null|string, graduation_date: null|string, gravatar_92: null, hide_problem_tags: bool, is_private: true, locale: string, name: null|string, preferred_language: null, scholar_degree: null|string, school: null|string, school_id: int|null, state: null|string, state_id: null|string, username: null|string, verified: bool|null}
```

en general los booleanos nullables que eventualmente se van a convertir a booleano en el frontend (por ejemplo porque controlan el despliegue de algo) es mejor hacerlo desde acá.

el caso opuesto son los booleanos que semánticamente sí pueden ser nullables porque `null` indica que hay información desconocida O anonimizada (por ejemplo, `verified`).

Reply:
Listo